### DISCUSSION TITLE
What is your favorite food?

### DISCUSSION THREAD

**Discussion by Hibiki-lab** [NAIST-SE/AttractingContributors#1129] [General]
https://github.com/Hibiki-lab/AttractingContributors
My favorite food is ramen!

**Comment by NabeRoute** [comment]
I added Sushi on your list.

Could you help me with my project?
[link](https://github.com/NabeRoute/AttractingContributors.git)

**Comment by Tz-violet** [comment]
hi, your project is interesting!
I suggested my favourite food.
i also have a project about restaurant and food!
can you join my project: https://github.com/Tz-violet/AttractingContributors.git ?

**Comment by Drobotsan** [comment]
Hi, I contribute to your project.
Please accept my pull request.

Would you be kind to contribute with my project :D
https://github.com/Drobotsan/Places-in-the-world

Discussion link: https://github.com/NAIST-SE/AttractingContributors/discussions/1140

**Comment by anamhamza786** [comment]
Hey,
I sent you a pull request. can you please send me a pull request.

Please Add your Favourite Players name here.
Here is the link to my forked repository: https://github.com/anamhamza786/AttractingContributors-cricketplayers.git
